src/article/shared/UploadTool.js

```diff
@@ -3,14 +3,17 @@ import { ToggleTool } from '../../kit'
 export default class UploadTool extends ToggleTool {
   renderButton ($$) {
     const button = super.renderButton($$)
-    const fileType = this.getFileType()
     const isMultiple = this.uploadMultiple()
     const input = $$('input').attr({
-      'type': 'file',
-      'accept': fileType
+      'type': 'file'
     }).ref('input')
       .on('change', this.onFileSelect)
 
+    if (!this.acceptAllFileTypes) {
```

Comment:
This looks like a regression. Either define it as a property, or call it as a function

Reply:
Spotted it, because accelAllFileTypes was never called.